rllib/env/tests/test_multi_agent_env.py

```diff
@@ -329,19 +329,25 @@ def compute_actions(
             def get_initial_state(self):
                 return [{}]  # empty dict
 
+            def is_recurrent(self):
+                return True
+
         ev = RolloutWorker(
             env_creator=lambda _: gym.make("CartPole-v1"),
             default_policy_class=StatefulPolicy,
             config=(
                 AlgorithmConfig().rollouts(
-                    rollout_fragment_length=5, num_rollout_workers=0
+                    rollout_fragment_length=5,
```

Comment:
This can be removed again I think

Reply:
@avnishn We are not setting enable_connectors=True in solved tests

Reply:
ok